projects/i18n/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@vcd/ui-i18n",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^8.2.9",
```

Comment:
I think the peer dependency needs to be a range that includes at least the version of Angular 6 that VCD 9.7 supports which is 6.x. 4.4 and beyond would give us compatibility with 9.5.